# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-mag/gcc34.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual sloppy code gcc >= 3.4.x discovers.

  - Rene Rebe <rene@exactcode.de>

--- gnome-mag-0.11.9/magnifier/magnifier.c	2004-11-02 15:21:40.000000000 +0000
+++ gnome-mag-0.11.9-fixed/magnifier/magnifier.c	2004-12-12 05:20:47.000000000 +0000
@@ -1109,6 +1109,9 @@
 }
 
 static void
+magnifier_init_window (Magnifier *magnifier, GdkScreen *screen);
+
+static void
 magnifier_set_property (BonoboPropertyBag *bag,
 			BonoboArg *arg,
 			guint arg_id,
